sw/device/silicon_creator/lib/manifest_def.h

```diff
@@ -11,13 +11,8 @@
 #include "sw/device/silicon_creator/lib/manifest.h"
 
 /**
- * Manifest definition.
- *
- * Declaration of the manifest that resides in the .manifest section. This
- * should be defined with known values, such as `code_start`, `code_end`, and
- * `entry_point`, populated. The remaining fields will be updated in the binary
- * by the signer tool.
+ * Gets the manifest of the current boot stage.
  */
-extern const volatile manifest_t kManifest OT_SECTION(".manifest");
+const manifest_t *manifest_def_get(void);
```

Comment:
I guess we can use `inline`? (it won't matter as much if we move to LTO)

Reply:
Not inline on purpose so that we don't expose the non-`const` `kManifest_`.